umibukela/models.py

```diff
@@ -53,3 +50,62 @@ class Partner(models.Model):
 
     def __str__(self):
         return "[ID: %s] %s" % (self.slug, self.short_name)
+
+
+class Site(models.Model):
+    name = models.CharField(max_length=200)
+    slug = models.CharField(max_length=200, primary_key=True)
```

Comment:
The slug should be unique, but we should definitely have a regular integer primary key. Otherwise referential integrity becomes a mission if the name (and hence slug) change.